### PR TITLE
style(rxButton): Add styles for finish & close button

### PR DESCRIPTION
**Pull request by glynnis**
Relates to https://jira.rax.io/browse/UXENCORE-231 and https://jira.rax.io/browse/EOD-273 -- new button style for "finish & close" with multi-screen modals.

screen shot 2015-04-09 at 11 18 01 am

### CODE REVIEW

**Comment by megalord** [comment]
The framework provides a `complete` footer state for simple use cases. Do you mind applying this style to the [template](https://github.com/rackerlabs/encore-ui/blob/master/src/rxModalAction/templates/rxModalFooters.html#L7)?  You can test it in the demo for `rxModalAction`, too.

**Comment by glynnis** [comment]
@megalord Ah, nice! Added.

**Comment by comeatmebro** [comment]
[Wraith UI diff](http://eb4d4842ea70055541c8-b6a3ba129b5eb76af13f4d17e4c1a53b.r51.cf5.rackcdn.com/890/gallery.html)

**Comment by parlarjb** [comment]
LGTM

File: src/rxButton/rxButton.less [hunk 1]
```diff
@@ -58,6 +58,21 @@
...
+        color: @buttonDefaultBg;
+        -webkit-box-shadow: inset 0 0 0 2px @buttonDefaultBg;
+        -moz-box-shadow: inset 0 0 0 2px @buttonDefaultBg;
+        box-shadow: inset 0 0 0 2px @buttonDefaultBg;
+        &:hover,
+        &:focus {
+            color: @buttonDefaultBgHover;
+            -webkit-box-shadow: inset 0 0 0 2px @buttonDefaultBgHover;
+            -moz-box-shadow: inset 0 0 0 2px @buttonDefaultBgHover;
+            box-shadow: inset 0 0 0 2px @buttonDefaultBgHover;
```
Comment (CITguy) [inline src/rxButton/rxButton.less]: These can use the `.box-shadow()` mixin.

Comment (glynnis) [inline src/rxButton/rxButton.less]: Dah! Thank you :cake: 

Corrected.

**Comment by comeatmebro** [comment]
[Wraith UI diff](http://eb4d4842ea70055541c8-b6a3ba129b5eb76af13f4d17e4c1a53b.r51.cf5.rackcdn.com/890/gallery.html)

**Comment by comeatmebro** [comment]
[Wraith UI diff](http://eb4d4842ea70055541c8-b6a3ba129b5eb76af13f4d17e4c1a53b.r51.cf5.rackcdn.com/890/gallery.html)

**Comment by parlarjb** [comment]
LGTM

**Pull request merged**